Sources/View+Rect.swift

```diff
@@ -12,7 +12,7 @@ import Foundation
 import UIKit
 
 extension UIView {
-    func getRect(keepTransform: Bool) -> CGRect {
+    public func getRect(keepTransform: Bool) -> CGRect {
```

Comment:
To be cleaner, I would move immediately `getRect` and `setRect` in UIView+PinLayout.swift

Reply:
Done